Config/Database.py

```diff
@@ -1,27 +1,27 @@
 from __future__ import annotations
 from datetime import datetime
 from xmlrpc.client import Boolean
-from Employee import INVALID_DATETIME, INVALID_STR, Employee, PERMISSION_LEVELS, INVALID_PATH
-from EmployeeContainer import EmployeeContainer, EmployeeAdmin, EmployeeOther, EmployeeSelf, adminFields
+from Employee.Employee import INVALID_DATETIME, INVALID_STR, Employee, PERMISSION_LEVELS, INVALID_PATH
+from Employee.EmployeeContainer import EmployeeContainer, EmployeeAdmin, EmployeeOther, EmployeeSelf, adminFields
 from pathlib import Path
+from Config.fetch_resource import fetch_resource
 import csv
 from tkinter import messagebox
 import configparser
 import dataclasses
-import config
 
 empToContainer = {'active': 'Active', 'address': 'Address', 'apartment': 'Apartment', 'bank_info': 'BankInfo', 'city': 'City', 'commission': 'Commission', 'country': 'Country', 'D_O_B': 'DOB', 'Dept': 'Dept', 'Emp_ID': 'EmpID', 'End_Date': 'EndDate', 'home_email': 'HomeEmail', 'home_phone': 'HomePhone', 'hourly': 'Hourly',
                   'name': 'Name', 'Office_email': 'OfficeEmail', 'office_phone': 'OfficePhone', 'hashed_password': 'Password', 'pay_method': 'PayMethod', 'Permission_level': 'PermissionLevel', 'permitted_lock_on': 'PermittedLockOn', 'route': 'Route', 'SS_num': 'SSNum', 'salary': 'Salary', 'Start_Date': 'StartDate', 'state': 'State', 'Title': 'Title', 'zip': 'Zip'}
 contToEmp = dict((empToContainer[k], k)for k in empToContainer)
 readcfg = configparser.ConfigParser()
 readcfg.optionxform = lambda optionstr: optionstr
 try:
-    with open('config.ini', 'r') as configfile:
+    with open(fetch_resource('./Resources/config.ini'), 'r') as configfile:
         readcfg.read_file(configfile)
 except (configparser.ParsingError, FileNotFoundError):
-    readcfg['TRANSLATION'] = empToContainer
+    readcfg['TRANSLATION'] = contToEmp
```

Comment:
using a config file like this causes the config file to not actually do anything, and cause `readcfg['TRANSLATION'].get(k.name, empToContainer[k.name])` to always use the fallback value, which is what it would get if the ini was created based on empToContainer. If this fixes a issue for you, then it is because your config.ini is set up incorrectly and should set to the default values from `readcfg['TRANSLATION'] = empToContainer`

Reply:
My config.ini file was setup by your lines of code, and we do want the translation to be going from container (database) to employee, because that is how you are using it later on in the code.

Reply:
Your config.ini was setup with `readcfg['TRANSLATION'] = contToEmp`, I tested it. I just generated a new config.ini with `readcfg['TRANSLATION'] = empToContainer` and attached it to this(renamed it to a txt because github won't let me attach a .ini).
[config.txt](https://github.com/david3214/Team_2_CS_2450-601/files/10126339/config.txt)
the ini file is supposed to be setup in the format: [Employee_field] = [corresponding name from imported csv]
I tested your branch, the ini file doesn't work, changing a database header to a different name then changing the ini to match results in the field with the changed name not being read. I will update the code for the ini so that it will complain about missing fields instead of using a default value.

Reply:
The dist/debug and dist/main were added to gitignore, as well as I fixed an issue causing build to still be tracked since it had already been added previously